platform/stm32f4xx/CMakeLists.txt

```diff
@@ -56,4 +56,28 @@ endif()
 target_compile_definitions(stm32f4xx PUBLIC
 	-DCONFIG_MAX_ISR_PRIORITY=${CONFIG_MAX_ISR_PRIORITY})
 
+# Pick console driver
+message(STATUS "STM32F4XX: Checking [CONFIG_USE_CONSOLE]...")
+if(${CONFIG_USE_CONSOLE})
+	if(NOT DEFINED CONFIG_CONSOLE_DEVICE_NUM)
+		message(FATAL_ERROR "CONFIG_CONSOLE_DEVICE_NUM must be set in order to use console")
```

Comment:
The CONFIG_CONSOLE_DEVICE_NUM is difficult to understand. I believe that we need a better way to address specific usart instance in future.